### ISSUE TITLE
automate population of roles section

### ISSUE THREAD

**Issue by grudelsud** [devcon-london/website#8]
presently to approve a submission you need to be an admin

an admin is defined in the `roles` collection with a document identified by user id and a field `admin: true`

values in the roles collection are added manually using the firebase console, there should be a way to do it from the website (e.g. if you are an admin, a button "make admin" is available in the members list page)

**Comment by silviopaganini** [comment]
revisiting this

wouldn't it be easier if the Member collection had a `admin: true` instead?

**Comment by grudelsud** [comment]
no, that wouldn't work. The problem is that you would give every user the permission to set themselves as admins.

**Comment by silviopaganini** [comment]
only if they had access to the DB on firebase... ?

**Comment by grudelsud** [comment]
they do have access to the database, as they can change their own data. this is the relevant rule: https://github.com/devcon-london/website/blob/master/firestore.rules#L18
